core/src/main/groovy/noe/common/utils/Java.groovy

```diff
@@ -10,11 +10,29 @@ import groovy.util.logging.Slf4j
 @Slf4j
 public class Java {
 
-  public static final String javaVersion = System.getProperty('java.version')
-  public static final String javaVendor = System.getProperty('java.vendor')
-  public static final String javaVmName = System.getProperty('java.vm.name')
-  public static final String javaVmInfo = System.getProperty('java.vm.info')
+  public static String javaVersion = System.getProperty('java.version')
+  public static String javaVendor = System.getProperty('java.vendor')
+  public static String javaVmName = System.getProperty('java.vm.name')
+  public static String javaVmInfo = System.getProperty('java.vm.info')
   public static final String javaHome = System.getenv('JAVA_HOME')
+  public static final String serverJavaHome = System.getenv('SERVER_JAVA_HOME')
+  private static final String javaHelperClassResource = "java/JavaVersion.java"
+
+  static {
+    if(serverJavaHome) {
```

Comment:
I don't see more elegant solution either. 

However, at that moment the identification executed with each static method calling, right? I would recommend some caching here.

Reply:
That's correct, I thought of that but thought that it was not worth to spend the time on. I have provided a basic boolean check; wdyt?

Reply:
I did not expect anything complicated either. I was thinking about caching per `serverJavaHome` though. However, I think current solution could be enough (at least for so time).

Thank you, +1 for merge